app/js/components/left_hand_nav/left_hand_nav.jsx

```diff
@@ -34,7 +36,15 @@ const onChange = () => {}
 const LeftHandNav = ({ leftHandNavVisible, dashboards, selectedDashboardId }) => (
   <Drawer open={leftHandNavVisible} className='left-hand-nav'>
     <LeftHandNavHeader imgPath={analyticsPath} text='Analytics' />
-    <Divider />
+    {
```

Comment:
What do you think about extracting this to a separate component? Too much?

Reply:
I think it makes sense when you reuse component. But this is just simple `if` statement and one link.